@here/harp-datasource-protocol/tsconfig.json

```diff
@@ -2,13 +2,27 @@
     "compilerOptions": {
         "target": "es2017",
         "module": "commonjs",
+        "outDir": "dist/cjs",
```

Comment:
I'm worried - not sure though that  that all code that imports directly from "@here/harp-*/lib/..." instead of index will fail.
And we do this internally:

```
$ grep -ER 'import .* from "@here/harp-.*/lib'. @here/harp-*/lib
@here/harp-datasource-protocol/lib/Theme.ts:import { Vector3Like } from "@here/harp-geoutils/lib/math/Vector3Like";
@here/harp-examples/lib/PerformanceUtils.ts:import { debugContext } from "@here/harp-mapview/lib/DebugContext";
@here/harp-geojson-datasource/lib/GeoJsonDecoder.ts:import { TileDecoderService } from "@here/harp-mapview-decoder/lib/TileDecoderService";
```

This will somehow work in our monorepo but my suspicion is that it will fail to find these files in workspace that contains only compiled installed by npm in node_modules.